### PR TITLE
Tag docker images with latest tag

### PR DESCRIPTION
**Pull request by gabrieljackson**
```release-note
None
```

### CODE REVIEW

File: scripts/push-docker.sh [hunk 1]
```diff
@@ -7,12 +7,25 @@ set -euox
...
 
-docker push mattermost/mattermost-cloud:$TAG
-docker push mattermost/mattermost-cloud-e2e:$TAG
+docker tag mattermost/mattermost-cloud:test mattermost/mattermost-cloud:$COMMIT_SHA
+docker tag mattermost/mattermost-cloud-e2e:test mattermost/mattermost-cloud-e2e:$COMMIT_SHA
+
+docker push mattermost/mattermost-cloud:$COMMIT_SHA
+docker push mattermost/mattermost-cloud-e2e:$COMMIT_SHA
+
+if [ "$REF_NAME" = "master" ] || [ "$REF_NAME" = "main" ]; then
```
Comment (fmartingr) [inline scripts/push-docker.sh:23]: Shouldn't this be `==`?

```suggestion
if [ "$REF_NAME" == "master" ] || [ "$REF_NAME" == "main" ]; then
```

**Review by fmartingr — COMMENTED** [review]
LGTM except for this one comment

**Review by mirshahriar — APPROVED** [review]

File: scripts/push-docker.sh (hunk 1 above)
Comment (mirshahriar) [reply scripts/push-docker.sh:23]: For single [, we can use =. For [[, we can use ==

Comment (gabrieljackson) [reply scripts/push-docker.sh:23]: Yeah, this is my understanding as well. For the record I hate shell scripting and could be wrong.

Comment (mirshahriar) [reply scripts/push-docker.sh:23]: https://linuxize.com/post/how-to-compare-strings-in-bash/#comparison-operators

Comment (fmartingr) [reply scripts/push-docker.sh:23]: My bash is a bit rusted as well, that's why I wasn't 100% sure.

Comment (fmartingr) [reply scripts/push-docker.sh:23]: Was chcking more information about it: https://unix.stackexchange.com/a/306115

**Pull request merged**